src/Response/GetLogResponse.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright (C) 2018 Adshares sp. z o.o.
+ *
+ * This file is part of ADS PHP Client
+ *
+ * ADS PHP Client is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * ADS PHP Client is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with ADS PHP Client.  If not, see <https://www.gnu.org/licenses/>
+ */
+
+namespace Adshares\Ads\Response;
+
+use Adshares\Ads\Entity\Account;
+use Adshares\Ads\Entity\EntityFactory;
+
+/**
+ * Response for GetLog request.
+ *
+ * @package Adshares\Ads\Response
+ */
+class GetLogResponse extends AbstractResponse
+{
+    /**
+     *
+     * @var Account
+     */
+    protected $account;
+
+    /**
+     * @var array
+     */
+    protected $log;
+
+    /**
+     *
+     * @param array $data
+     */
+    protected function loadData(array $data): void
+    {
+        parent::loadData($data);
+
+        if (array_key_exists('account', $data)) {
+            $this->account = EntityFactory::createAccount($data['account']);
+        }
+        if (array_key_exists('log', $data)) {
+            $log = $data['log'];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "$log" which has the same name as the field declared at line 42. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1117)